DESCRIPTION

```diff
@@ -5,6 +5,7 @@ Authors@R: c(
     person("Hugo", "Gruson", , "hugo.gruson+R@normalesup.org", role = c("aut", "cre", "cph"),
            comment = c(ORCID = "0000-0002-4094-1476")),
     person("Thibaut", "Jombart", role = "aut"),
+    person("Carmen", "Tamayo", role = "aut"),
```

Comment:
Please also add your ORCID.

Reply:
I added it here https://github.com/epiverse-trace/episoap/pull/100/commits/abb336c3587b6523c7a59dbde456dc606a3c5204 in the same way that Hugo had, please let me know if this is correct

Reply:
Yes, looks correct.